src/wasm-emscripten.h

```diff
@@ -29,6 +29,8 @@ void generateMemoryGrowthFunction(Module&);
 // signature in the indirect function table.
 std::vector<Function*> makeDynCallThunks(Module& wasm, std::vector<Name> const& tableSegmentData);
 
+void addGlobalImports(Module& wasm, std::unordered_set<Name> const& globalImports);
```

Comment:
(And of course the fact that we use wasm globals to implement that is an implementation detail and also makes the term 'global' more confusing :))